package.json

```diff
@@ -59,8 +59,8 @@
   "husky": {
     "hooks": {
       "commit-msg": "commitlint -E HUSKY_GIT_PARAMS",
-      "pre-push": "yarn build-tests && yarn run lint && yarn run test-changed",
-      "pre-commit": "yarn verify-commit && yarn split-e2e-tests && pretty-quick --staged"
+      "pre-push": "yarn build-tests-changed && yarn split-e2e-tests",
```

Comment:
Doesn't this change mean the output of `split-e2e-tests` will not be committed?

Reply:
The output is already not committed. This script only runs as a precheck to validate that the base config is valid. The real config is generated by CCI on the fly